code-rules.tex

```diff
@@ -16,7 +16,16 @@
 
 \section*{Abstract}
 
-We present ten tips to help researchers ensure that
+In open source, there is always a risk that a project will lose its maintainers,
+with the grimly named "bus factor" typically described as a risk factor.
```

Comment:
```suggestion
with the grimly named "[bus factor](https://en.wikipedia.org/wiki/Bus_factor)" typically described as a risk factor.
```

We could link to Wikipedia as an external resource, the article also links to a number of other resources but I didn't investigate those further. 

I think a link would be useful for those who want a bit more information.